spec/collectionspace/converter/core/collectionobject_spec.rb

```diff
@@ -9,9 +9,10 @@
     '/document/*/objectNumber',
     '/document/*/numberOfObjects',
     '/document/*/titleGroupList/titleGroup/title',
-    { xpath: '/document/*/titleGroupList/titleGroup/titleLanguage',  transform: ->(text) { CSURN.parse(text)[:label] } },
+    { xpath: '/document/*/titleGroupList/titleGroup/titleLanguage',  transform: ->(text) { text.gsub!(/urn:.*?item:name\([^)]+\)'([^']+)'/, '\1') } },
```

Comment:
@mark-cooper Is there a better way to make the tests pass when multiple URN values get returned? It seems that using `CSURN.parse(text)[:label]` in this proc only works if there's a single XML element returned. What I've got here works, but seems pretty clunky. I'm not real familiar with working with procs like this.

Reply:
@kspurgin you can use an index for the xpath to test one element (or more) specifically, for example:

https://github.com/collectionspace/cspace-converter/blob/master/spec/collectionspace/converter/anthro/osteology_spec.rb#L26-L27

For xpath first position is 1 not 0.

Reply:
Thanks, @mark-cooper ! Now get back to being on vacation. :-D

Reply:
My comments or mentions are just to make sure I don't forget them and do not indicate the need for any immediate attention :-)